### ISSUE TITLE
Block Theme: A quick design review / Design QA

### ISSUE THREAD

**Issue by jasmussen** [WordPress/pattern-directory#633]
This explores the block theme conversion. Looks really good, you can take the bits of feedback here that make the most sense, and if anything requires more conversation, can skip that.

Small things:

* Block Pattern Directory → Patterns
* “Search patterns” instead of “Search patterns…”
* Copy pattern could have a min-width so it doesn’t change size when it turns to “Copied pattern”. May even consider changing it to “Copied!” instead of “Copied pattern”.

To consider:

* Rephrase the tagline. Maybe: “Copy and paste ready-to-go layouts into any WordPress site.”
* Use [the pattern bar from here](https://www.figma.com/file/bz4qs4jZCkqvQzPH2fmX6C/Pattern-Directory?type=design&node-id=4844%3A6087&mode=design&t=VNZubffNjfZnMHNk-1) instead of the new dropdown categories.

Individual pattern permalinks

* EB Garamond headings

Can use a smaller corner radius:

![corner-radius](https://github.com/WordPress/pattern-directory/assets/1204802/dcec4b4d-4024-43ab-9e4d-d21044590458)

**Some designs**

Outline and solid stars for favorite and favorited:

![stars](https://github.com/WordPress/pattern-directory/assets/1204802/84d93d48-5d7b-4df2-b909-5c43bb8257a0)

**Footer:**

![footer](https://github.com/WordPress/pattern-directory/assets/1204802/27dd0e4e-d289-430a-950a-b8c614349399)

To consider, depending on how much we want to do, can do at any time.

* “Learn more” instead of the more verbose sentence.
* Secondary outline buttons for both, don’t need the “Create” to have that white emphasis. There’s also a “New pattern” in the header.
* Consider reducing the two paragraphs both, there’s a lot of text.
* Either find an editorial way to have the same amount of lines (e.g. 3 lines of text), or consider applying a min-height to the paragraphs so the buttons line up.

This entire footer can also be ignored for now, and be re-thought separately.

Favorites/My Patterns

* There’s a bit more [here](https://www.figma.com/file/bz4qs4jZCkqvQzPH2fmX6C/Pattern-Directory?type=design&node-id=4846%3A52548&mode=design&t=VNZubffNjfZnMHNk-1).
* Use EB Garamond for the heading.
* Can hide search.
* The “Declined” or “Pending” chips can be overlayed, and differently colored, like this from [here](https://www.figma.com/file/bz4qs4jZCkqvQzPH2fmX6C/Pattern-Directory?type=design&node-id=4849%3A3045&mode=design&t=VNZubffNjfZnMHNk-1):

![pending](https://github.com/WordPress/pattern-directory/assets/1204802/06449ebb-fbf0-4ddb-a2ce-535d95514f61)

**New pattern**

There’s a brief flash of unstyled text before the editor loads. Can we hide that, perhaps even show a spinner?

Probably a followup, but there are some designs for how to potentially improve the create pattern flow [here](https://www.figma.com/file/bz4qs4jZCkqvQzPH2fmX6C/Pattern-Directory?type=design&node-id=4846%3A53827&mode=design&t=VNZubffNjfZnMHNk-1).

**Comment by ryelle** [comment]
I've addressed most of this [in the PR](https://github.com/WordPress/pattern-directory/pull/624), so I'm going to close the issue.

> Rephrase the tagline. Maybe: “Copy and paste ready-to-go layouts into any WordPress site.”

Leaving this out for now since it's not a final decision— if we want to do a full copy review, that can happen any time.

> Use [the pattern bar from here](https://www.figma.com/file/bz4qs4jZCkqvQzPH2fmX6C/Pattern-Directory?type=design&node-id=4844%3A6087&mode=design&t=VNZubffNjfZnMHNk-1) instead of the new dropdown categories.

This is phrased as a suggestion here, but since this would require a bit more custom code, I'd like to figure out if it's a hard requirement before spending the time.

> Either find an editorial way to have the same amount of lines (e.g. 3 lines of text), or consider applying a min-height to the paragraphs so the buttons line up.

Since the container is flexible, a min-height would cause other issues— if we did a 5-line min-height it would look really strange when both containers are 4 lines (at 1600 wide). You could try working with marketing to get 3 lines of text at some size, but the container will still be responsive.

> New pattern

The pattern creator is separate from the theme, but I do have a follow-up PR ready to go that will fix this once the theme is merged.

Everything else should be addressed now. If there's anything else, please open separate issues for each problem.

**Comment by jasmussen** [comment]
> This is phrased as a suggestion here, but since this would require a bit more custom code, I'd like to figure out if it's a hard requirement before spending the time.

I think it might be, I'd love @richtabor's input as he was a big part of pushing for the initial setup.

Thank you for all the work!

**Comment by jasmussen** [comment]
@ryelle this may already be fixed but not deployed, but just making sure to capture it, this title should also just say "Patterns", not "Block Pattern Directory":

![Screenshot 2024-03-14 at 10 43 23](https://github.com/WordPress/pattern-directory/assets/1204802/5822fab2-9740-4e6d-8afb-8724b2655bbf)

**Comment by ryelle** [comment]
Nope, that's set elsewhere, so it was missed. This issue is closed though, can you drop that into a new issue?

**Comment by richtabor** [comment]
> Use [the pattern bar from here](https://www.figma.com/file/bz4qs4jZCkqvQzPH2fmX6C/Pattern-Directory?type=design&node-id=4844%3A6087&mode=design&t=VNZubffNjfZnMHNk-1) instead of the new dropdown categories.

> This is phrased as a suggestion here, but since this would require a bit more custom code, I'd like to figure out if it's a hard requirement before spending the time.

> I think it might be, I'd love @richtabor's input as he was a big part of pushing for the initial setup.

Catching up here, but yes a row of categories is good.
